__mocks__/vitals.mock.ts

```diff
@@ -827,7 +827,7 @@ export const mockVitalData = [
     diastolic: 72,
     pulse: 22,
     temperature: 37,
-    oxygenation: 30
+    oxygenSaturation: 30
```

Comment:
Pedantry unrelated to the topic of this PR: I standardized all the vitals vocabulary to match CIEL preferred terms.